frontend/src/component/project/Project/ProjectFeatureToggles/ProjectFeatureToggles.tsx

```diff
@@ -265,6 +266,20 @@ export const ProjectFeatureToggles = ({
                           Cell: FeatureTagCell,
                           width: 80,
                           searchable: true,
+                          filterName: 'tags',
+                          filterBy(
+                              row: IFeatureToggleListItem,
+                              values: string[]
+                          ) {
+                              return includesFilter(
+                                  getColumnValues(this, row),
+                                  values
+                              );
+                          },
+                          filterParsing(value: string) {
+                              // only first tag from the list is added to search suggestions
+                              return value.split('\n')[0];
```

Comment:
when returning filtering hints we return only first tag if we have more than one